crates/bevy_reflect/bevy_reflect_derive/src/container_attributes.rs

```diff
@@ -48,9 +48,13 @@ impl TraitImpl {
     /// Returns whichever value is not [`TraitImpl::NotImplemented`].
     /// If both values are [`TraitImpl::NotImplemented`], then that is returned.
     /// Otherwise, an error is returned if neither value is [`TraitImpl::NotImplemented`].
```

Comment:
This need updating. The documentation is out of date now. Changing this to an in-place mutation is a good idea btw.